sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -78,13 +78,21 @@ export interface CreateQueueOptions extends OperationOptions {
     userMetadata?: string;
 }
 
+// @public
+export interface CreateRuleOptions {
+    action?: SqlRuleAction;
+    filter?: SqlRuleFilter | CorrelationRuleFilter;
+    name: string;
+}
+
 // @public
 export interface CreateSubscriptionOptions extends OperationOptions {
     autoDeleteOnIdle?: string;
     availabilityStatus?: EntityAvailabilityStatus;
     deadLetteringOnFilterEvaluationExceptions?: boolean;
     deadLetteringOnMessageExpiration?: boolean;
     defaultMessageTimeToLive?: string;
+    defaultRuleOptions?: CreateRuleOptions;
```

Comment:
I could have gone with an overload instead. But the new overload looked messy with three different kinds of option bags.

```ts
```

Hence making it an option inside the existing `CreateRuleOptions`.